app/src/main/java/com/example/pokedex/api/RetrofitInstance.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.pokedex.api
+
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitInstance {
+
+    private const val BASE_URL = "https://pokeapi.co/api/v2/"
```

Comment:
create a constants file and move this BASE_URL to it

Reply:
Ok. Created Utils package and object to store base url

![BaseURL](https://github.com/Obehi234/Pokedex/assets/48982222/71e384ef-aeac-41ef-bc92-96b754b70aac)